File: Examen1/2026_1/PlantillaC#/TestSoft/TestSoftPersistance/Alumnos/DAO/AlumnoDAO.cs
using TestSoftModel.Alumno;
using TestSoftPersistance.DAO;

namespace TestSoftPersistance.Alumnos.DAO
{
    public interface AlumnoDAO : IDAO<Alumno>
    {
        // Patron SELECT INTO: busca por codigo y retorna el id (0 si no existe)
        int BuscarPorCodigo(string codigo);
    }
}

File: Examen1/2026_1/TestSoft/TestSoftPersistance/src/main/java/pe/edu/pucp/testsoft/persistance/alumno/dao/AlumnoDAO.java
package pe.edu.pucp.testsoft.persistance.alumno.dao;

import pe.edu.pucp.testsoft.persistance.dao.IDAO;
import pe.edu.pucp.testsoft.model.alumno.Alumno;

public interface AlumnoDAO extends IDAO<Alumno> {
    // Patron SELECT INTO: busca por codigo y retorna el id (0 si no existe)
    int buscarPorCodigo(String codigo);
}
